Index: cmake/internal_utils.cmake
===================================================================
--- cmake/internal_utils.cmake   (revision 660)
+++ cmake/internal_utils.cmake   (working copy)
@@ -66,6 +66,9 @@
       # Resolved overload was found by argument-dependent lookup.
       set(cxx_base_flags "${cxx_base_flags} -wd4675")
     endif()
+    if (MSVC_VERSION EQUAL 1700)
+      set(cxx_base_flags "${cxx_base_flags} -D_VARIADIC_MAX=10")
+    endif ()
     set(cxx_base_flags "${cxx_base_flags} -D_UNICODE -DUNICODE -DWIN32 -D_WIN32")
     set(cxx_base_flags "${cxx_base_flags} -DSTRICT -DWIN32_LEAN_AND_MEAN")
     set(cxx_exception_flags "-EHsc -D_HAS_EXCEPTIONS=1")
